index.js

```diff
@@ -78,6 +109,34 @@ class TrezorKeyring extends EventEmitter {
     return this.model;
   }
 
+  /**
+   * Gets the vendor, if known.
+   *
+   * @returns {"trezor" | "onekey"}
+   */
+  async getVendor() {
+    return this.vendor || (this.vendor = await this.fetchVendor());
+  }
+
+  /**
+   * fetch vendor by call getFeatures
+   * @param {object} features
+   * @returns {'onekey' | 'trezor' | null}
+   */
+  fetchVendor() {
+    return TrezorConnect.getFeatures()
```

Comment:
Instead of using `then`/`catch` syntax, what do you think about using `async`/`await` syntax?

Reply:
There are actually calls to the TrezorConnect API in this repo in both styles, but I think it's good to use the newer standard. So I fixed it. 👍